docusaurus-plugin-moonwave/src/index.js

```diff
@@ -3,6 +3,86 @@ const fs = require("fs")
 const { promisify } = require("util")
 const exec = promisify(require("child_process").exec)
 
+const mapLinks = (nameSet, items) =>
+  items.map((name) => {
+    if (!nameSet.has(name)) {
+      throw new Error(
+        `Moonwave plugin: "${name}" listed in classOrder option does not exist`
+      )
+    }
+
+    return {
+      type: "link",
+      href: `/api/${name}`,
+      label: name,
+    }
+  })
```

Comment:
Extracted this logic out into a function - it was repeated 3 times in the file, now appearing just once!